### DISCUSSION TITLE
Outlining Next Season - Planning

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#2881] [Ideas]
I would like to share with you all a document I have been preparing that will outline how next season could work. **This is just my proposal after gathering some feedback from community, nothing set in stone or official.** I would like to hear people's thoughts and ideas around it. I also wanted to start nice and early, almost 2 months before next season would be scheduled to start.
___

Sunflower Land has a large variety of players, it has:

- Whales who like to drop $ to get items
- Competitive players who are willing to set alarms to get the special item 
- Players who HATE setting alarms in order to compete for top items
- Players who just want to be a part of the season by being able to participate on a low scale, potentially get a few items
- Players who don't care about the season but still want to feel included somehow
- Players from ALL over the world!
- Many more..

**SO..How can we make the season as fun and challenging for as many people as we can?** 

I have laid out a plan below, it aims to set up different avenues for players to participate in capturing as many of the core groups in the game. Keeping RNG in 1 place (TI). PLayers can choose their adventure/preference of how to traverse the season!
___

### System A: Auctions

Auctions for SFL and/or resources.

### System B: Chores & Deliveries

Chores from Hayseed Hank and Deliveries from a variety of NPCs would reward the player tickets. NO CROP RNG. I have provided a few options for how Chores could work:

_Option 1_
Chores are no longer a set per day, but 1 appearing at a time (Similar to season 1). When a chore is completed, the next one appears right after. As a player acquires enough tickets that the item asks for, they can mint it. Players who grind the most/hardest and are able to fulfill most deliveries will acquire the item first until it runs out.
[Option to have a 'preview' for the next 3 chores]
[Big advantage to those with powerful items; Wings, LH etc... could make those chores appear less often? Still an advantage]

_Option 2_
5+ Chores are generated every day at reset as they are currently for WE but only for 5 days in a row, on the 6th and 7th day there are no chores. The 6th day will be a rest day or a day for unlucky timezone players to catch up on. On the 7th day, feather style auctions will occur. 8th day the cycle continues.
[Player ties would be common, winner decided by xp?]

_Option 3_
Every day 3 Chores are generated at 00UTC and 3 chores are generated at 12UTC (Thx Blastaman for idea), on the 7th day every week there are no chores (Auction Day).
On the 7th day, run 2-4 auctions for an item. As there are no chores for this day, it does not matter if a player wakes up earlier/later than another. Does this solve the EU issue?
[Player ties would be common, winner decided by xp?]

**Deliveries will work the same as they currently do across all options.**

### System C: TI RNG

Introduce a new resource to be found at TI, set aside some supply of the items requiring the new TI resource to be available for minting. TI would be aligned with the season completely, removing the extra workload of having to come up with new items alongside the season.

### System D: Gameplay RNG

_Option 1_
Each action on a farm will award players with a very small chance of awarding them with Flower Tickets (Different from the chore/delivery tickets). Actions such as harvesting crops, chopping trees or mining minerals. Weighted by value of action (As it is currently with crops) 
In the last week of the season, Items are put up for auction requiring Flower Tickets. 
[Too confusing? Too many tickets?]

_Option 2_
Players receive RARE Raffle tickets by performing actions on farm. Raffle is held at end of season
[Chance of bots/multis winning]

___

So how would it all work with supply of items?

Supply of an item is split across the different 'Systems', an example would be as such:

![image](https://github.com/sunflower-land/sunflower-land/assets/108110065/f6649319-95ad-409e-8510-3c0afa655d94)

TI supply could be even lower and chore could be higher.. a player can only mint 1 item  across ALL Systems (Minting item x in TI means you wont be able to mint it for the raffle or chore)
___

- Are you a whale that just wants to drop X amount of SFL and resources for an item without dropping a sweat? SFL/Resource auctions are for you!
- Are you a grindy player that wants to compete versus others for the top items? Chores/Deliveries are for you!
- Do you LOVE RNG and TI? Well off to Treasure Island you go! 
- Do you want to participate in the season but not want the stress or spend as much as whales whilst still having the chance to get items? Gameplay RNG would suit you :D

Still thinking about where to fit Bert..

I appreciate your time for reading if you made it this far, I will also appreciate any suggestions or feedback to how you think this type of season would work, any positives or negatives.. general comments etc.

**Just a reminder, this is NOT FINAL, it is simply a DRAFT i compiled after talking with a few community members, reading a lot of comments and threads on Discord and personally taking part in the season.**

Cheers :)

**Comment by vitt0** [comment]
I like this proposal a lot. I think it addresses many of the issues we currently face in this season and the previous ones too, mostly concerning the grind versus non-grind aspects. I believe this is one of the things that divide player feedback on what to expect from the season.

That said, when it comes to System B, I prefer option 1 the most. I don't like the other two options because if ties become too common, the ticket system may lose its importance, and XP would become the main deciding factor.

You mentioned that, for example, Wings and Luna's Hat would have an advantage. To somewhat mitigate their impact, I think it would be good to follow your suggestion of reducing the number of tasks where they provide an advantage. Additionally, I'd like to suggest that we consider using numbers where Wings don't offer such a significant advantage. For example:

- Harvest 40 Radish

This seems like a good example where Wings most likely won't provide an advantage since the odds of getting 40 instant crops in one round of Radish are pretty low. I understand that we can't balance all the tasks, but it's something to keep in mind when deciding the required numbers, mostly for the long crops like Radish and above. As for Luna's Hat, I don't think there's a way to "balance" it other than reducing the frequency of cooking tasks and avoiding duplicates, like having a task that says "Cook 1 Carrot Cake" instead of "Cook 3 Carrot Cakes." This would help reduce the "extra hours someone skips."

One last thing I'd like to mention is that I believe the list of tasks would work better if they were randomly selected from a pool of tasks, with some safeguards in place. For example, not allowing the same task to appear twice in a row. I'm primarily thinking about Eggs and Gold tasks, where having them back-to-back could put players at a significant disadvantage.

I also really like the idea of having a "preview" of the next 3 tasks or something similar. It would help players plan ahead, similar to how we had it in Season 1 when everyone followed the list of tasks. However, since it's only three tasks, it wouldn't mean that players could prepare for everything.

**Reply by IamSooy** [reply]
Never to System B: Option 1!

Current Chores & Deliveries are still better!

**Comment by rodolfoddias** [comment]
Maybe the tasks preview could be a feature for banner holders

**Comment by Chucrutty** [comment]
System E

to contemplate Bert, you could make Bert tickets and have a small supply to mint via collector

**Comment by Nandario** [comment]
I completely agree with you, the game is made up of several classes of players, and the BERT update which was not scheduled from the start led us to a false track, the season has become impossible to predict and the top leaderloard is decided by luck and not consistency as it was announced at the start, if Bert does not decide to display an NFT that you have and the date of the auction has come you lose your bid and your effort and your token burning without getting anything in return, I believe that the updates during the season are strongly discouraged, fell from the TOP 100 leaderboard to 200 because of BERT was not on my agenda, especially since I 've burned everything I had, in order to mint a rare NFT, luck was not on my side and losing the race because of luck is frustrating especially when it is not announced from the start

**Comment by snorri-snorri** [comment]
Deliveries really need to work like the way they did at the start of the season where you could see what was coming 24hrs ahead.

Yes players should have supplies of everything before an event but eventually everything starts to run low, a delivery essentially just becomes a chore but you don't keep the end product.

Balancing time and tasks is key during an event, constantly supplying 40-50 eggs and bulk amount of slow growing crops while making cakes and cooking/eating food for chores too is a difficult thing to do when you can't see what best to plan for until 6hrs after they were announced (unless you wake up at 0000UTC).

Worrying about the advantage that Wings or LH might give is ok but imagine how much harder you have to farm without GA to meet the delivery requirements.
Not saying the advantage shouldn't be there but don't make it even harder for those players without the powerful NFTs by having future deliveries only available to be seen at 0000 UTC.

**Comment by Cato234** [comment]
Every time system b option 1 this season 0 competition the only difference lucky rng and bert which causes the nft to increase the price inorganically and after 3 days it goes to the floor. Wings is easy to solve the problem just make don't count for quests the instant crops but LH very difficult, maybe just make decent supply like 200 the best nfts next season and never again low supply like IP or LH.

**Comment by PapyMax** [comment]
For the chores, neutralize Wings, why not, but neutralize all items/advantages (MP, Obie, LH, Cabbage Girl, kernaldo, skills etc).

Why some Items can give an avantage for chores and not the others ? Who decide the avantage is enough big to be disabled ? People who has not the item ? Or it's random ? Or it only because wing's holder are vilain ?

Or we can simply remove from the chore, harvest parsnip, corn and cabbage ? or any cooking (because of LH).
More the game will go, more it will be complicated (impossible) to make something exactly balancedfor everybody.

In fact, it's already too late, there is nothing in the game which is not changed by any boost if we consider the skill. So how we do to make the chores totaly balanced for everybody ? Just by neutralising the wings ? Or reducing the number of cake we have to cook (IMO which has no effect on the advantage of LH).

**Comment by Blasta-alasta** [comment]
Obviously I like the idea. I'll drop the link of my suggestions i posted a while ago to https://github.com/sunflower-land/sunflower-land/discussions/2837

**Comment by ghost** [comment]
I kind of like the current time limited items, everyone can mint them but still, not everyone is minting them for a better one or for other items. this in turn make it so that the supply is kept in check, the reason is there are so many options that could be minted using feathers, like the 3000 feather wings, other decor items, other wearables and auction by feathers.

people generally want a chance to compete for the top tier items like this season's LH and IP which only the top players were able to, specially new players, so this needs to be addressed if you want to include everyone. this would also means that doing this, would make the season more grindy like what happened during the cabbage girl season.

on the other hand, being a top player required one to put large amount of funds and time in the game to be one which is an advatage of the top players or early players.

for me avoiding IP and LH tier items should be avoided being released in seasons and should be released on seprate events instead where everyone has a chance to compete.

there maybe some holes in this, but still it is a concern of some players every season too

**Comment by Ivan-Ricardo-Lopes** [comment]
If Bert remains in the next season, I'm not even going to bother. I'll just complete the daily quests and collect the minor rewards of the season. This Bert feature mainly benefits those who have the money to afford the high prices, as well as those who trade quickly—buying items as soon as a Bert item is revealed and then selling them moments later.

**Comment by Blasta-alasta** [comment]
You know I’m a massive supporter of uncapped chores (as it was with s1/2 )for the seasons…. 

But the more I think about it, the more I think the current system is better (or a variation of it, Option 2/3 in your proposal).

Apart from the issue from the EU portion of players about reset time, this season's chores have definitely gotten more interesting the last couple of weeks. At first they were too easy but now it’s seems like the pressure has been put on to complete the ‘hard’ chore everyday…. Which is good imo…

And it stops the issue of select SFTs giving to much of an advantage… eg with uncapped there is nothing to stop those with wings/luna's hat/mysterious parsnip/etc soaring ahead. I do wonder if the day of rest is needed as suggested, as the pressure of completing these hard chores could be what separates those at the top of LB instead of RNG or EXP.

**Comment by Blasta-alasta** [comment]
For System D, the Gameplay RNG.... aimed at the more newer players, I have a couple of ideas:

**Raffle Ticket Drops**
What if during the season all players could have a chance to find a raffle ticket just from regular planting (could include chopping and mining too, encouraging more activity). The raffle tickets could be exchanged at a new NPC house to enter one of a series of raffles. So if a player finds raffle ticket they can choose which raffle they want to enter. Multiple tickets could be used for multiple entries.

**Feed NPC**
Same idea with the raffle tickets and the raffle house except maybe there is a very Hungry NPC that needs feeding everyday (or multiple times a day) to get the tickets. To help new players maybe it could only include food items from Fire Pit & Kitchen. I'm thinking since it's the X-Mas Season it could be a reindeer maybe.... thats eats more than just reindeer carrots lol.

**Comment by Aminbtdds** [comment]
First of all thanks for great discussion as always😍🍻❤
I waited for major auctions of season to be finished in order to give my feedback for next season planning, we have several types of players, we can identify several types of players as Dcol said👇
you've identified several types of players in Sunflower Land:

1. **Whales**: These are players who are willing to spend a significant amount of money to acquire in-game items or advantages.

2. **Competitive Players**: These players are highly motivated to compete for top items and are even willing to set alarms or invest extra effort to do so.

3. **Casual Players**: Some players prefer a more relaxed gaming experience and may not want to set alarms or engage in intense competition but still want to participate.

4. **Inclusive Players**: This group values being part of the season and the community but may not be focused on obtaining top-tier items.

5. **Indifferent Players**: Players who might not care much about the season but still want to feel included or participate in some way.

6. **Global Player Base**: Recognizing that your player base comes from all over the world, with different time zones and preferences.

By considering these 5 player types and global player base, we aim to create a season that caters to a wide range of player preferences and playstyles, which can enhance overall player satisfaction and retention.

Imho, In order to do that first of all we need to know each type's numbers(which devs can do that by monitoring players activity or by running surveys in game) and proportion to allocate supply in relevant.
Based on poll we ran on our telegram community, we had 2% whales, 25% competitive, 57% casual, 8% inclusive and 8% indifferent players.(these percents would be different considering all players worldwide).

We should allocate reasonable supply of SFTs for each group:
1. Whales: Allocate a portion of the supply to whales, acknowledging their investment and support for the game. This group might receive a smaller number of supply due to their financial advantage and smaller number of them.
2. Competitives: Reward competitive players with a significant portion of the SFTs supply. These players are actively engaged in the game and contribute to its competitiveness.
3. Casuals: Allocate a reasonable number of supply to casual players, ensuring they feel valued and included. Casual players may not invest as much time as competitive players, but they are still an essential part of the community and game.
4. Inclusives: Dedicate a portion of the supply to inclusive players. These players may not be highly competitive, but they actively participate in community events and support the game in various ways.
5. Indifferents: Consider allocating a small number of SFTs to indifferent players to encourage their participation and engagement. Providing them with a taste of the game's benefits might spark their interest.
6. Three things to consider about supply beside player types:
A. Global player base: fairness about different timelines.
B. Early adopters: their loyalty and support should get recognised.
C. Commiunity contributors: Recognize their efforts in fostering a positive community environment.

Also, When dividing the supplies, it's essential to strike a balance between recognizing different player types and ensuring that each group feels appreciated. Transparent communication about the allocation process can also help manage expectations and foster a sense of fairness within the community.
Additionally, consider reserving a small portion of the supply for future community events.

Personally i don't think WE season SFTs supply distribution was right. Allocating half of top tier nfts supply in blind auction with token(it is just like allocating half of the supplies for whales) will result in unwanted side effects that are not healthy:

1. **Economic Disparity**: The in-game economy might become significantly stratified, with a small group of players (whales) possessing a substantial portion of valuable SFTs. This can create economic disparity, potentially leading to a less balanced and less fair gaming environment.

2. **Player Frustration**: Non-whale players, especially competitive and casual players, might feel frustrated or discouraged if a significant number of top SFTs are inaccessible to them. This frustration can be exaggerated when RNGs are involved which could lead to decreased player engagement and satisfaction.

3. **Whale Dominance**: The game dynamics might be heavily influenced by the decisions and actions of whales, potentially shaping the gameplay experience in ways that favor their interests. This could lead to a lack of diversity in strategies and playstyles, reducing overall game depth.

4. **Market Imbalance**: If a large portion of Tier 1 NFTs is concentrated in the hands of a few whales, the in-game marketplace could become less dynamic. With fewer NFTs circulating among a wider player base, trading and market activities might be limited.

5. **Community Perception**: The gaming community might perceive the game as pay-to-win or heavily favoring players with substantial financial resources. This perception can impact the game's reputation and player acquisition in the long run.

6. **Social Dynamics**: The social aspects of the game, such as player collaboration and community building, might be affected if a significant portion of players feels excluded or marginalized due to the dominance of whales.

7. **Monetary Impact**: While the blind auction might generate revenue from whale participants in form of token burning, there could be unintended consequences if the overall player base diminishes in size or engagement due to the perceived imbalance.

To mitigate these challenges, game developers might consider strategies such as limiting the number of Tier 1 NFTs available in the blind auction for whales, implementing caps on individual purchases, or introducing alternative methods for non-whale players to obtain Tier 1 NFTs. Balancing the economic aspects of the game while ensuring a fair and engaging experience for all players is essential for the long-term success and sustainability of the game.

Hope my feedback will be considered in supply allocation of next season's SFTs🍻❤🌻

**Reply by saba-hosseiny** [reply]
کامل و بی نقص
خوشخالم که‌جزو بازیکنان رقابتی سرسخت هستیم🥺🥺
